libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -127,7 +138,11 @@
     <ng-content
       select=".sky-input-group-icon.sky-input-box-icon-inset-left"
     ></ng-content>
-    <ng-container *ngTemplateOutlet="hostIconsInsetLeftTemplate"></ng-container>
+    <ng-container *ngIf="hostIconsInsetLeftTemplate">
```

Comment:
☝️ Why are these `*ngIf` checks needed?

Reply:
Because they come in via the `populate` method and thus the type of the properties had to change to `TemplateRef | undefined` and `ngTemplateOutlet` does not allow `undefined` and so the template won't compile without the `ngIf` surrounding it.